### DISCUSSION TITLE
Should we release 0.70.5?

### DISCUSSION THREAD

**Discussion by dmytrorykun** [reactwg/react-native-releases#43] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/44

---

Current Release: [0.70.4](https://github.com/facebook/react-native/releases/tag/v0.70.4)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
we need this https://github.com/facebook/react-native/commit/0e316ec671617f5e7c1985b4b05cd0d45bcea403

**Comment by retyui** [comment]
a small one but helpful: https://github.com/facebook/react-native/commit/475310dbbaec8048411edefc6cdddab330df7966

**Comment by matthew-dean** [comment]
Yes, please bring flex-gap sooner! 🙏

**Reply by cortinico** [reply]
Hiding this as offtopic as you're not following the pick request described in the top post:

> Conversations on this thread are limited:
>    [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
>    [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).
> Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

**Comment by kelset** [comment]
bump of CLI https://github.com/react-native-community/cli/releases/tag/v9.3.0

**Reply by leotm** [reply]
that was [fast](https://github.com/leotm/react-native-template-new-architecture/pull/1154) 🔥 just checking its fine since causing a [CMake error](https://github.com/leotm/react-native-template-new-architecture/actions/runs/3393245928/jobs/5640357880) atm on Android
but ofc i'm on unsupported JDK 19 and Gradle 7.6rc (CI still on NDK 21.4, locally NDK 25.1)

```console
FAILURE: Build failed with an exception. * What went wrong:
Execution failed for task ':app:configureCMakeDebug[arm64-v8a]'.
> [CXX1405] error when building with cmake using /home/runner/work/react-native-template-new-architecture/react-native-template-new-architecture/android/app/src/main/jni/CMakeLists.txt: Build command failed.
  Error while executing process /usr/local/lib/android/sdk/cmake/3.18.1/bin/cmake with arguments {.../android/app/src/main/jni -DCMAKE_SYSTEM_NAME=Android -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_SYSTEM_VERSION=21 -DANDROID_PLATFORM=android-21 -DANDROID_ABI=arm64-v8a -DCMAKE_ANDROID_ARCH_ABI=arm64-v8a ...
```

i'll test further this wkend and from vanilla RN 0.70.4

edit: nvm couldn't repro and CI failing for unrelated reason now (looks like 9.3.1's out too)

**Comment by kelset** [comment]
remove `"@jest/create-cache-key-function": "^29.0.3",` from package.json, no idea how it ended up in the cherry pick (https://github.com/facebook/react-native/commit/f5ebb692ab9b3ca40fb1dc79c5902f04d04f7a55) when it wasn't in the PR (https://github.com/facebook/react-native/pull/35022/files)

**Reply by leotm** [reply]
i think (idk if still worth keeping)
- https://github.com/facebook/react-native/pull/35014
> It's used in the jest preset

**Comment by cortinico** [comment]
Here the PR to include the fix for #35210 inside the next point release of 0.70:
- https://github.com/facebook/react-native/pull/35216
